src/main/java/appeng/me/storage/MEMonitorIInventory.java

```diff
@@ -73,7 +73,7 @@ public void removeListener( final IMEMonitorHandlerReceiver<IAEItemStack> l )
 	}
 
 	@Override
-	public IAEItemStack injectItems( final IAEItemStack input, final Actionable type, final BaseActionSource src )
+	public IAEItemStack injectItems( final IAEItemStack input, final Actionable type, final IActionSource src )
 	{
 		ItemStack out = ItemStack.EMPTY;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "out". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1854)